### PR TITLE
Revert "Hide Vertical tab strip even when it's browser fullscreen"

### PR DESCRIPTION
**Pull request by sangwoo108**
Reverts brave/brave-core#19802

Fix https://github.com/brave/brave-browser/issues/33106

### CODE REVIEW

**Comment by sangwoo108** [comment]
Hi, @goodov . This would be new to you, but could you take a look at this? Simon is on PTO.

**Review by simonhong — APPROVED** [review]
++
It seems `Simone` is on PTO this week not me :)

**Comment by sangwoo108** [comment]
@simonhong Thanks! haha I thought it was you

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 11 x64` using the following build(s):

```
Brave | 1.60.37 Chromium: 117.0.5938.88 (Official Build) nightly (64-bit)
-- | --
Revision | d869c8860226f51bdef9f6a46f18f89cd2ae656c
OS | Windows 11 Version 22H2 (Build 22621.2283)
```

Basically ensure that vertical tabs are visible/usable when in `Fullscreen Mode` using `F11`. On `Win`, there's currently no way of viewing the horizontal tabs while in `Fullscreen` which is known and will be addressed in another fix.

* ensured that vertical tabs are visible and usable while you're in `Fullscreen Mode`

image

----------------------

Verification **PASSED** on `macOS

```
Brave | 1.60.38 Chromium: 117.0.5938.88 (Official Build) nightly (x86_64)
--- | ---
Revision | 89f7ea7a01a3ab15163916a9441c473ab1021372
OS | macOS Version 13.5.2 (Build 22G91)
```

Basically ensure that vertical tabs are visible/usable when in `Fullscreen Mode` using `fn + f`. Also ensured that horizontal tabs are also visible/usable as per the following:

**`Example`** | **`Example`**
---------------|-------------------
Screenshot 2023-09-19 at 9 25 58 PM | Screenshot 2023-09-19 at 9 30 17 PM
